test/runtests.jl

```diff
@@ -1,33 +1,70 @@
-using Distributed, Test, CUDA
+using Distributed, Test, CUDA, Pkg
+
+using AdvancedHMC: AdvancedHMC
 
 println("Envronment variables for testing")
 println(ENV)
 
+const DIRECTORY_AdvancedHMC = dirname(dirname(pathof(AdvancedHMC)))
+const DIRECTORY_Turing_tests = joinpath(DIRECTORY_AdvancedHMC, "test", "turing")
+const GROUP = get(ENV, "GROUP", "All")
```

Comment:
Can we name `GROUP` more explict, e.g. `AHMC_TEST_GROUP`?

Reply:
Sure, I was just following the rest of the Turing-ecosystem, but I'm whatever about it :+1:

Reply:
Yeah I was worrying about potential conflicts with other testing suites so I thought an explict name might be helpful.